### ISSUE TITLE
Debian repository signature cannot be verified

### ISSUE THREAD

**Issue by tcrass** [dbeaver/dbeaver#36492]
### Description

When executing the steps mentioned on https://dbeaver.io/download/ for installation from a repository on a Debian Testing system, apt complains

```
Err:19 https://dbeaver.io/debs/dbeaver-ce  InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY D33A1BD725ED047D
Warning: https://dbeaver.io/debs/dbeaver-ce/InRelease: The key(s) in the keyring /usr/share/keyrings/dbeaver.gpg.key are ignored as the file has an unsupported filetype.
Warning: GPG error: https://dbeaver.io/debs/dbeaver-ce  InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY D33A1BD725ED047D
Error: The repository 'https://dbeaver.io/debs/dbeaver-ce  InRelease' is not signed.
Notice: Updating from such a repository can't be done securely, and is therefore disabled by default.
Notice: See apt-secure(8) manpage for repository creation and user configuration details.
```

I don't know if something has recently changed how Debian treats repository keys, but if so, the instructions should be updated accordingly.

### DBeaver Version

Community 24.3

### Operating System

Debian Testing

### Database and driver

---

### Steps to reproduce

```
sudo  wget -O /usr/share/keyrings/dbeaver.gpg.key https://dbeaver.io/debs/dbeaver.gpg.key
echo "deb [signed-by=/usr/share/keyrings/dbeaver.gpg.key] https://dbeaver.io/debs/dbeaver-ce /" | sudo tee /etc/apt/sources.list.d/dbeaver.list
sudo apt-get update && sudo apt-get install dbeaver-ce
```

**Comment by mayerro** [comment]
@tcrass 
you can try this instructions 
```
sudo mkdir -p /usr/share/keyrings/
curl -s https://dbeaver.io/debs/dbeaver.gpg.key | sudo gpg --dearmor -o /usr/share/keyrings/dbeaver.gpg.key
```
instead of 
```
sudo  wget -O /usr/share/keyrings/dbeaver.gpg.key https://dbeaver.io/debs/dbeaver.gpg.key
```

**Comment by Koratsuki** [comment]
> @tcrass you can try this instructions
> 
> ```
> sudo mkdir -p /usr/share/keyrings/
> curl -s https://dbeaver.io/debs/dbeaver.gpg.key | sudo gpg --dearmor -o /usr/share/keyrings/dbeaver.gpg.key
> ```
> 
> instead of
> 
> ```
> sudo  wget -O /usr/share/keyrings/dbeaver.gpg.key https://dbeaver.io/debs/dbeaver.gpg.key
> ```

Thanks a lot, you saved my day, fixed it.

**Comment by tcrass** [comment]
Can confitm that this works; the trick seems to be processing the downloaded file through gpg. Also works with wget, of course.
Could you please update the instructions on the website?

Cheers -- tcrass

**Comment by arhayka** [comment]
@tcrass, @Koratsuki thank you for testing!
We updated the instruction on https://dbeaver.io/download/

**Comment by samstride** [comment]
@arhayka , the website now has the old instructions which causes the `... the public key is not available: NO_PUBKEY ...` error.

Instruction on website as of `25.0.4`:

```sh
sudo  wget -O /usr/share/keyrings/dbeaver.gpg.key https://dbeaver.io/debs/dbeaver.gpg.key
echo "deb [signed-by=/usr/share/keyrings/dbeaver.gpg.key] https://dbeaver.io/debs/dbeaver-ce /" | sudo tee /etc/apt/sources.list.d/dbeaver.list
sudo apt-get update && sudo apt-get install dbeaver-ce
```
Changing the first command to:

```sh
curl -s https://dbeaver.io/debs/dbeaver.gpg.key | sudo gpg --dearmor -o /usr/share/keyrings/dbeaver.gpg.key
```

... resolves the issue.

**Comment by marchrius** [comment]
Today had same problem as @samstride, visited dbeaver.io download page and followed instructions but without success. Then I landed here after googled error and solution is exactly this https://github.com/dbeaver/dbeaver/issues/36492#issuecomment-2856072342

Please, update the instructions on that page.

**Comment by mzatko** [comment]
Err:22 https://dbeaver.io/debs/dbeaver-ce  InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 132C13A8A330F403
